### PR TITLE
DBZ-5792: Fix for - Message with LSN XYZ not present among LSNs seen …

### PR DESCRIPTION
**Pull request by rajdangwal**
Fixes: https://issues.redhat.com/browse/DBZ-5792

### CODE REVIEW

**Comment by jpechane** [comment]
@rajdangwal Warning, I've rebased on the latest main and force-pushed to bypass the CI Postgres test failure

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgoutput/PgOutputMessageDecoder.java [hunk 1]
```diff
@@ -130,6 +130,17 @@ public boolean shouldMessageBeSkipped(ByteBuffer buffer, Lsn lastReceivedLsn, Ls
         try {
             MessageType type = MessageType.forType((char) buffer.get());
             LOGGER.trace("Message Type: {}", type);
+            switch (type) {
+                case TYPE:
+                case ORIGIN:
+                case TRUNCATE:
```
Comment (jpechane) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgoutput/PgOutputMessageDecoder.java:136]: Plese check pgoutput CI results. The `TRUNCATE` message is supported by Debezium.

**Review by jpechane — CHANGES_REQUESTED** [review]
Please check pgoutput CI results. The `TRUNCATE` message is supported by Debezium.

**Comment by rajdangwal** [comment]
Thanks @jpechane for taking care of the build.
The required tests are passing now.

**Comment by jpechane** [comment]
@rajdangwal Applied, thanks!

**Pull request merged**